include/dali/core/error_handling.h

```diff
@@ -224,12 +224,19 @@ inline dali::string GetStacktrace() {
 #define DALI_STR(x) DALI_STR2(x)
 #define FILE_AND_LINE __FILE__ ":" DALI_STR(__LINE__)
 
-#define DALI_MESSAGE(str)\
+#define DALI_MESSAGE_WITH_STACKTRACE(str)\
   (std::string("[" FILE_AND_LINE "] ") + str + dali::GetStacktrace())
 
+#define DALI_MESSAGE(str)\
+  (std::string("[" FILE_AND_LINE "] ") + str)
+
 #define DALI_FAIL(str)                            \
-    throw dali::DALIException(DALI_MESSAGE(str)); \
+    throw dali::DALIException(DALI_MESSAGE_WITH_STACKTRACE(str));
 
+#define DALI_ERROR(str)                            \
```

Comment:
Either align the backslashes
```
if (a)      \
   asdf     \
```
or move them to one space after end of line
```
if (a) \
    asdf \
```

Reply:
Will do in #1280